wkcuber/pyproject.toml

```diff
@@ -7,29 +7,15 @@ readme = "README.md"
 license = "AGPL-3.0"
 repository = "https://github.com/scalableminds/webknossos-libs"
 homepage = "https://docs.webknossos.org/wkcuber"
-include = ["wkcuber/version.py"]  # included explicitly since it's in .gitignore
+
+[tool.poetry.scripts]
+wkcuber = "wkcuber.main:app"
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-attrs = "^22.1.0"
-cattrs = "^22.2.0"
-cluster_tools = { path = "../cluster_tools/", develop = true }
-czifile = "^2019.7.2"
+typer = {extras = ["all"], version = "^0.9.0"}
 GitPython = "^3.0.5"
```

Comment:
What is this used for?

Reply:
I removed it, it was previously used in _internal/versioning.py